libs/payments/paypal/src/lib/paypal.manager.ts

```diff
@@ -10,24 +10,58 @@ import {
   StripeInvoice,
   StripeManager,
 } from '@fxa/payments/stripe';
-import { AccountDatabase } from '@fxa/shared/db/mysql/account';
 import { PayPalClient } from './paypal.client';
 import { BillingAgreement, BillingAgreementStatus } from './paypal.types';
-import {
-  PaypalCustomerNotFoundError,
-  PaypalCustomerMultipleRecordsError,
-} from './paypalCustomer/paypalCustomer.error';
+import { PaypalCustomerMultipleRecordsError } from './paypalCustomer/paypalCustomer.error';
 import { PaypalCustomerManager } from './paypalCustomer/paypalCustomer.manager';
+import { PaypalManagerError } from './paypal.error';
 
 @Injectable()
 export class PayPalManager {
   constructor(
-    private db: AccountDatabase,
     private client: PayPalClient,
     private stripeManager: StripeManager,
     private paypalCustomerManager: PaypalCustomerManager
   ) {}
 
+  /**
+   * Create billing agreement using the ExpressCheckout token.
+   *
+   * If the call to PayPal fails, a PayPalClientError will be thrown.
+   */
+  public async createBillingAgreement(token: string): Promise<string> {
+    const response = await this.client.createBillingAgreement({
+      token,
+    });
+    return response.BILLINGAGREEMENTID;
+  }
```

Comment:
Will be merging this with Meghan's PR (https://github.com/mozilla/fxa/pull/16630). Whichever goes in first, I'll rebase the other.